### ISSUE TITLE
Deploy doesn't work

### ISSUE THREAD

**Issue by Galibov** [ivarvermeulen/nuxt-serverless-at-edge#4]
Hi, I tried deploy your example, and get this errors:
1.  First I get "bucketname already exist" I changed
` before:deploy:deploy: 
       - aws s3 cp --recursive ./.nuxt/dist/client/ s3://nuxt-serverless-at-edge/_nuxt/ --cache-control "max-age=31536000,public,immutable"
       - aws s3 cp --recursive ./static/ s3://nuxt-serverless-at-edge/ --exclude "README.md" --cache-control "max-age=31536000,public,immutable"`

To this:

` after:deploy:deploy: 
       - aws s3 cp --recursive ./.nuxt/dist/client/ s3://nuxt-serverless-at-edge/_nuxt/ --cache-control "max-age=31536000,public,immutable"
       - aws s3 cp --recursive ./static/ s3://nuxt-serverless-at-edge/ --exclude "README.md" --cache-control "max-age=31536000,public,immutable"`

2. After fixed first error i get this: 
ERROR

The request could not be satisfied.

The Lambda function returned an invalid request or response to CloudFront. 
Generated by cloudfront (CloudFront)

Please, can you update configuration? Thanks!

**Comment by ivarvermeulen** [comment]
s3 bucket names need to be unique, so you need to alter the name of the bucket 'nuxt-serverless-at-edge' to something unique, you will need to alter that name for every occurrence in the serverless.yaml file

**Comment by Galibov** [comment]
Yes, I know it and put unique bucket name, but still it doesn't work.

**Comment by ivarvermeulen** [comment]
Are you still getting a deploy error or and error when requesting the Cloudfront url?

**Comment by Galibov** [comment]
After I change "before:deploy:deploy" on "after:deploy:deploy"  I start get error when requesting to the Cloudfront url?. I used your example from this repo, just changed bucketname and deploy queue

**Comment by provderek** [comment]
Hey Ivar, thanks for the demo...trying to get our Nuxt project running serverless, and based on your demo, tried to get running in lambda. If I just build a complete skeleton project with nuxt create app, and only add in serverless, I already hit the 50MB Lambda function limit. Serverless uses aws api which is like 60MB or somethng alone. In a real production project with many libraries in node modules, how can we possibly run serverless in lambda with their limits? What am I missing? Thanks
